haystack/nodes/prompt/invocation_layer/hugging_face.py

```diff
@@ -2,12 +2,13 @@
 import logging
 import os
 
+from transformers import PreTrainedTokenizer, PreTrainedTokenizerFast, PreTrainedModel
```

Comment:
These imports should be moved to the `with LazyImport` block, this should fix the failing linting check.

Reply:
thanks! will update!